sharepoint-server/xml/Microsoft.SharePoint.Taxonomy/Term.xml

```diff
@@ -332,6 +332,7 @@ namespace Microsoft.SDK.SharePoint.Samples
         <remarks>
           <para>The copied <b>Term</b> will have the same properties as the current <b>Term</b>, except for the <see cref="P:Microsoft.SharePoint.Taxonomy.Term.Name" /> property. If the term is copied with its child terms, the newly created child terms will have the same properties as the child terms of the current term including the Name property.The copied child terms will perserve the same hierarchy.</para>
           <para>The current user must have <b>TaxonomyRights.EditTerm</b> permission in order to use this method. Call the <see cref="M:Microsoft.SharePoint.Taxonomy.TermStore.CommitAll" /> method to save this change to the database.</para>
+          <para>From mid-May 2020, the name of the copied term would be the name of the original term appended with "- Copy", instead of being the name of the original term prefixed with "Copy of". This will allow the original and the copies of a term to be listed together when sorted alphabetically. Multiple copies of the same term would be suffixed by a serial number starting from 1.</para>
```

Comment:
```suggestion
          <para>From mid-May 2020, the name of the copied term is the name of the original term appended with "- Copy", instead of being the name of the original term prefixed with "Copy of". This allows the original and the copies of a term to be listed together when sorted alphabetically. Multiple copies of the same term will be suffixed by a serial number starting from 1.</para>
```